android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerEvent.kt

```diff
@@ -51,5 +47,17 @@ class RNGestureHandlerEvent private constructor() : Event<RNGestureHandlerEvent>
       (EVENTS_POOL.acquire() ?: RNGestureHandlerEvent()).apply {
         init(handler, dataExtractor)
       }
+
+    @JvmStatic
```

Comment:
Do we need `@JvmStatic` annotation here?

Reply:
I assumed we do because `obtain` method above is also annotated as `JvmStatic`. I have checked and it's working fine without them, should they be removed in this case?

Reply:
Yes, it's probably leftover from rewrite when `obtain` was called from the yet-to-be rewritten Java class.

Reply:
I removed them in c9e43c7078e07a205ff9ac8378d61e6ee9b4fbc6